lib/ansible/modules/system/facter.py

```diff
@@ -39,13 +39,18 @@
 
 def main():
     module = AnsibleModule(
-        argument_spec=dict()
+        argument_spec = dict(
+            external_dir = dict(required=False, type='str')
```

Comment:
`facter` accepts multiple `external_dir` at a time. So we can make this is as a `list` rather than `str`.

Reply:
Also, add documentation for this new parameter.